### PR TITLE
Added 2.0.0 references to the Manage Airflow Versions doc

### PR DESCRIPTION
**Pull request by jwitz**
Updated all 2.0 references on this page: https://www.astronomer.io/docs/cloud/stable/customize-airflow/manage-airflow-versions 

I also swapped the table columns for Alpine and Debian-based images since we no longer support Alpine.

FYI - There's not an official Apache blog for 2.0.0. yet, so I currently have 2.0 references linked to Astronomer's own blog: https://www.astronomer.io/blog/introducing-airflow-2-0 Lmk if we should ice this until the official release!

### CODE REVIEW

File: cloud/stable/04_customize-airflow/01_manage-airflow-versions.md [hunk 1]
```diff
@@ -152,16 +153,16 @@ For our platform's full collection of Docker Images, reference [Astronomer on Qu
...
+| [v1.10.7](https://github.com/astronomer/ap-airflow/blob/master/1.10.7/CHANGELOG.md)   | FROM quay.io/astronomer/ap-airflow:1.10.7-buster-onbuild  | FROM quay.io/astronomer/ap-airflow:1.10.7-alpine3.10-onbuild  |
+| [v1.10.10](https://github.com/astronomer/ap-airflow/blob/master/1.10.10/CHANGELOG.md) | FROM quay.io/astronomer/ap-airflow:1.10.10-buster-onbuild | FROM quay.io/astronomer/ap-airflow:1.10.10-alpine3.10-onbuild |
+| [v1.10.12](https://github.com/astronomer/ap-airflow/blob/master/1.10.12/CHANGELOG.md) | FROM quay.io/astronomer/ap-airflow:1.10.12-buster-onbuild | ROM quay.io/astronomer/ap-airflow:1.10.12-alpine3.10-onbuild  |
+| [v1.10.14](https://github.com/astronomer/ap-airflow/blob/master/1.10.14/CHANGELOG.md) | FROM quay.io/astronomer/ap-airflow:1.10.14-buster-onbuild | N/A                                                           |
+| [v2.0.0](https://www.astronomer.io/blog/introducing-airflow-2-0)                      | FROM quay.io/astronomer/ap-airflow:2.0.0-buster-onbuild   | N/A                                                           |
```
Comment (paolaperaza) [inline cloud/stable/04_customize-airflow/01_manage-airflow-versions.md:163]:
```suggestion
| [v2.0.0](https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md)                      | FROM quay.io/astronomer/ap-airflow:2.0.0-buster-onbuild   | N/A                                                           |
```

Sort of confusing, but basically in this table I link out to the AC changelog (in `ap-airflow`) and in the section above where we talk about Airflow versions I link out to the OSS release notes or blog.

File: cloud/stable/04_customize-airflow/01_manage-airflow-versions.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Read below for details.
 
 Astronomer Certified offers support for the following versions of Apache Airflow:
 
+- [Airflow 2.0.0](https://www.astronomer.io/blog/introducing-airflow-2-0)
```
Comment (paolaperaza) [inline cloud/stable/04_customize-airflow/01_manage-airflow-versions.md:27]: This is a good place-holder, but I confirmed with Ash he'll be publishing an OSS blog post with 2.0 changes - so once that's out, we can link here!

**Comment by paolaperaza** [comment]
Awesome, thanks @jwitz ! Can you update that ap-airflow link? Once that's done, I'll approve and we can wait to merge this until game-time Thursday morning + add the OSS 2.0 blog post as well.

File: cloud/stable/04_customize-airflow/01_manage-airflow-versions.md (hunk 1 above)
Comment (jwitz) [reply cloud/stable/04_customize-airflow/01_manage-airflow-versions.md]: Updated this in latest commit!

**Review by paolaperaza — APPROVED** [review]
Let's rock!

**Pull request merged**